docs/dictionary/property/currentCard.lcdoc

```diff
@@ -0,0 +1,36 @@
+Name: currentCard
+
+Type: property
+
+Syntax: the currentCard
+
+Summary:
+A stack property that is set to the short name of the card in a given stack that is selected for display.
+
+OS: mac, windows, linux, ios, android
+
+Platforms: desktop, server, mobile
+
+Example:
+ -- the following results in the same value as put the short name of the current card of this stack
+ 
+put the currentCard of this stack
+
+ -- even if the stack is invisible, in the background, or offscreen
+
+set the visible of this stack to false
+
+put currentCard of this stack
+
+-- can be retrieved or set for a stack file that is not currently opened (provided the stack file is writeable)
+
+set currentCard of stack "/Users/admin/Desktop/example.livecode" to "ExampleOfCardNameThatsNotTheFirstCard"
+
+Description:
+<currentCard> <property> is a stack property that is set to the short name of the card that is currently selected
+
+for display in a given stack, not neccessarly the front most stack nor necessarily a visible stack or even open stack.
+
+References: current card (glossary)
+
+Tags: card, stack, property, this, current
```

Comment:
These should be references, they are not tags (and you don't need `current` as it is a synonym of `this`)

I'm not sure if there is a tag that currently applies - maybe `navigation`?